tests/test_registration.py

```diff
@@ -50,6 +50,7 @@ def test_registration(entry_point):
     assert len(all_specs) == 2
     assert all_specs[0].id == "SomeEntity-v0"
     assert all_specs[1].id == "SomeEntity-v1"
+    repr(all_specs[0])
```

Comment:
generally im against doing things just for the sake of getting 100% coverage, but in this case this is actually doing something: if we borked something in the repr then this would raise an error. so maybe just a comment above like "Check that the repr doesn't raise any errors" to be clear that while we're not testing the output is as we expect (cause that would be boring as hell) we are actually doing something worthwhile

Reply:
Yea, it was the coverage that pointed out what wasn't being tested, and it looked a line which could (in theory) break, so thought it was worth adding (and to get that 100% lol). 

I wasn't feeling stoked about just adding a rando `repr(...` into that test anyways, so I did it properly and explicitly now. Lmk what you think